patroni/api.py

```diff
@@ -293,6 +293,16 @@ def do_GET_metrics(self):
         metrics.append("# TYPE patroni_dcs_last_seen gauge")
         metrics.append("patroni_dcs_last_seen{0} {1}".format(scope_label, postgres.get('dcs_last_seen', 0)))
 
+        metrics.append("# HELP patroni_pending_restart Value is 1 if cluster need a restart, 0 otherwise.")
```

Comment:
s/need/needs/ I think

Reply:
You're right, it's fixed :)

Reply:
Not really, the `pending_restart` flag is per member (node).

Reply:
Oh ok. I thought it was for the cluster after changing PostgreSQL settings, but make sense for nodes.

Reply:
Hi ! The sentence was fixed, let me know if it's ok for you.